app/pdfs/timelogs_pdf.rb

```diff
@@ -0,0 +1,21 @@
+require 'prawn'
+require 'prawn/table'
+class TimelogsPdf < Prawn::Document
+
+  def initialize(timelogs)
+    super()
+    @timelogs = timelogs
+    fill_document
+  end
+
+  def fill_document
```

Comment:
if the method is used only inside object it is better to do it `private` (encapsulation).